frontend/app/work_packages/view_models/parent-relations-handler.js

```diff
@@ -28,7 +28,10 @@
 
 module.exports = function(CommonRelationsHandler, WorkPackageService, ApiHelper) {
   function ParentRelationsHandler(workPackage, parents, relationsId) {
-      var handler = new CommonRelationsHandler(workPackage, parents, relationsId);
+      var relations = parents.filter(function(parent) {
```

Comment:
Call me curious, but why does the parameter `parents` contain the work package in the first place?

To me this looks like the parameter passed is already invalid, isn't it?